### PR TITLE
Backtracking Algorithms.

### PR DESCRIPTION
**Pull request by SubhradeepSS**

### CODE REVIEW

**Comment by SubhradeepSS** [comment]
@PragatiVerma18  please review my PR #65

**Comment by SubhradeepSS** [comment]
@Bhavnaharitsa   please review my PR

File: Algorithms/Search/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Beginner
```
Comment (Bhavnaharitsa) [inline Algorithms/Search/README.md]: Why so many commits?

**Comment by Bhavnaharitsa** [comment]
This is not a good practise to add so many commits for one pr and every commit has only one line modification...Do all the necessary changes and then commit
Closing this.. reduce the number of commits make sensible commits and re open

**Pull request closed without merging**